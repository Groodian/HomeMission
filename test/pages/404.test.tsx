import Custom404 from '../../src/pages/404';
import { render } from './testUtils';

jest.mock('next/router', () => require('next-router-mock'));

describe('404 page', () => {
  it('matches snapshot', async () => {
    const { asFragment } = await render(<Custom404 />);
    expect(asFragment()).toMatchSnapshot();
  });

  it('matches snapshot DE', async () => {
    const { asFragment } = await render(<Custom404 />, { language: 'de' });
    expect(asFragment()).toMatchSnapshot();
  });
});
