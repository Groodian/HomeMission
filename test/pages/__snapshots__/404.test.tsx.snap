// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 page matches snapshot 1`] = `
<DocumentFragment>
  .emotion-0 {
  text-align: center;
  width: 300px;
  margin-left: auto;
  margin-right: auto;
}

.emotion-1 {
  margin: 0;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 400;
  font-size: 3rem;
  line-height: 1.167;
  letter-spacing: 0em;
}

.emotion-2 {
  margin: 0;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  border-width: 0;
  border-style: solid;
  border-color: rgba(0, 0, 0, 0.12);
  border-bottom-width: thin;
}

.emotion-3 {
  margin: 0;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 400;
  font-size: 1.5rem;
  line-height: 1.334;
  letter-spacing: 0em;
}

<div
    class="MuiBox-root emotion-0"
  >
    <h1
      class="MuiTypography-root MuiTypography-h3 emotion-1"
    >
      404
    </h1>
    <hr
      class="MuiDivider-root MuiDivider-fullWidth emotion-2"
    />
    <h3
      class="MuiTypography-root MuiTypography-h5 emotion-3"
    >
      This page could not be found.
    </h3>
  </div>
</DocumentFragment>
`;

exports[`404 page matches snapshot DE 1`] = `
<DocumentFragment>
  .emotion-0 {
  text-align: center;
  width: 300px;
  margin-left: auto;
  margin-right: auto;
}

.emotion-1 {
  margin: 0;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 400;
  font-size: 3rem;
  line-height: 1.167;
  letter-spacing: 0em;
}

.emotion-2 {
  margin: 0;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  border-width: 0;
  border-style: solid;
  border-color: rgba(0, 0, 0, 0.12);
  border-bottom-width: thin;
}

.emotion-3 {
  margin: 0;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 400;
  font-size: 1.5rem;
  line-height: 1.334;
  letter-spacing: 0em;
}

<div
    class="MuiBox-root emotion-0"
  >
    <h1
      class="MuiTypography-root MuiTypography-h3 emotion-1"
    >
      404
    </h1>
    <hr
      class="MuiDivider-root MuiDivider-fullWidth emotion-2"
    />
    <h3
      class="MuiTypography-root MuiTypography-h5 emotion-3"
    >
      Diese Seite konnte nicht gefunden werden.
    </h3>
  </div>
</DocumentFragment>
`;
